osc/commandline.py

```diff
@@ -6621,12 +6624,11 @@ def parse_repoarchdescr(self, args, noinit = False, alternative_project = None,
     def do_build(self, subcmd, opts, *args):
         """${cmd_name}: Build a package on your local machine
 
-        You need to call the command inside a package directory, which should be a
-        buildsystem checkout. (Local modifications are fine.)
+        The command works from a package checkout (local changes are fine).
 
-        The arguments REPOSITORY and ARCH can be taken from the first two columns
-        of the 'osc repos' output. BUILD_DESCR is either a RPM spec file, or a
-        Debian dsc file.
+        You can use `osc repos` to look up REPOSITORY and ARCH arguments. If
+        they are not set, `osc` choses suitable at random. BUILD_DESCR is
```

Comment:
I don't think it's good to say the repository is picked at random.

The order of repositories is deterministic, it's just not documented how the repositories are ordered, specifically.

Reply:
Fair enough. Documented the exact order.